### PR TITLE
update: clickhouse dictionaries now named collections is released

### PR DESCRIPTION
**Pull request by joelynch**
https://aiven.atlassian.net/browse/DDB-1150

## Checklist

- [ ] The first paragraph of the page is on one line.
- [ ] The other lines have a line break at 90 characters.
- [ ] I checked the output.
- [ ] I applied the [style guide](../styleguide.md).
- [ ] My links start with `/docs/`.

### CODE REVIEW

File: docs/products/clickhouse/howto/create-dictionary.md [hunk 1]
```diff
@@ -82,13 +79,11 @@ with two exceptions,`ssd_cache` and `complex_key_ssd_cache`, which are not suppo
 - HTTP(s)
 - Upstream/remote ClickHouse
 - Aiven for ClickHouse
-
-<!--
 - Upstream/remote MySQL®
 - Aiven for MySQL
 - Upstream/remote PostgreSQL®
 - Aiven for PostgreSQL
--->
```
Comment (aris-aiven) [inline docs/products/clickhouse/howto/create-dictionary.md:91]: Change `Upstream/remote` to just `Remote`

**Review by aris-aiven — DISMISSED** [review]
Tiny wording simplification

File: docs/products/clickhouse/howto/create-dictionary.md (hunk 1 above)
Comment (joelynch) [reply docs/products/clickhouse/howto/create-dictionary.md]: maybe external would be better

Comment (aris-aiven) [reply docs/products/clickhouse/howto/create-dictionary.md]: Hesitated on external, will it be easily perceived easily as "external to the platform" vs. remote which is clearly "outside the node"?

**Review by wojcik-dorota — APPROVED** [review]
LGTM

**Pull request merged**